### PR TITLE
ICU-20568 Pull CLDR-13488's unit conversion data into units.txt

### PR DESCRIPTION
**Pull request by hugovdm**
##### Checklist

- [X] Issue filed: https://unicode-org.atlassian.net/browse/ICU-20568
- [X] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

**Comment by hugovdm** [comment]
Do we have an ICU ticket number for this yet?

(Found these related tickets:
https://unicode-org.atlassian.net/browse/ICU-20568
https://unicode-org.atlassian.net/browse/ICU-20920.)

**Comment by sffc** [comment]
https://unicode-org.atlassian.net/browse/ICU-20568 is the ticket we should use.

File: icu4c/source/data/misc/units.txt [hunk 1]
```diff
@@ -1,54 +1,321 @@
...
+            factor{"1"}
+            target{"kilogram-meter-per-square-second"}
+        }
+        ohm{
+            factor{"1"}
+            target{"kilogram-square-meter-per-cubic-second-square-ampere"}
+        }
+        one{
+            factor{"1"}
+            target{"1"}
```
Comment (sffc) [inline icu4c/source/data/misc/units.txt:215]: We should make the target `"one"`, not `"1"`, but that's a bug in the XML data, not in this PR.

**Review by sffc — APPROVED** [review]
Nice!

File: icu4c/source/data/misc/units.txt (hunk 1 above)
Comment (hugovdm) [reply icu4c/source/data/misc/units.txt:215]: As discussed, we'll do this in a follow-up pull request.

**Pull request merged**